### DISCUSSION TITLE
Running PKHeX with Bottles (Method)

### DISCUSSION THREAD

**Discussion by hyenatown** [kwsch/PKHeX#4336] [General]
I recently got PKHeX running with Bottles on multiple systems (with one little variation between them). I'd like to know if this works as described on other systems too.

In a newly created bottle:
- Using the `Application` template
- Runner set to `sys-wine-9.0` **_or_** `soda-9.0-1 (test)`
- DXVK set to `dxvk-2.4`
- VKD3D set to `vkd3d-proton-2.13`
- Windows version set to `Windows 7`
- (Optional - if for some reason a DPI detection issue occurs) Under Advanced Display Settings, enabling Virtual Desktop and setting the resolution as desired (1280x720 recommended)

Making sure that:
- Flatpak permissions are set to allow the user directory

My tests have pretty much concluded the application is stable!

**Comment by PixelsPerMinute** [comment]
Thanks for the solution!

With these settings, it also seems to run well on my own system.

You just need to make sure you also have the required .NET version installed within the same Bottle as PKHex for it to run. At least in my case.

**Comment by hyenatown** [comment]
I forgot to mention, the virtual desktop is not a requirement by any means, but for some reason my Radeon-based laptop was very stubborn about running the bottle without it. Despite also testing on my Nvidia-based desktop and Steam Deck...

**Comment by Wolf-lbh** [comment]
I am trying to do this but I get the following error: (Any help would be incredibly appreciated!)
`002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
0024:err:module:fixup_imports_ilonly mscoree.dll not found, IL-only binary L"System.Runtime.dll" cannot be loaded
0024:err:module:fixup_imports_ilonly mscoree.dll not found, IL-only binary L"System.Runtime.dll" cannot be loaded
0024:err:module:find_forwarded_export module not found for forward 'icuuc68.u_charsToUChars_68' used by L"C:\\windows\\system32\\icu.dll"
Cannot get symbol u_charsToUChars from libicuuc
Error: 127`
